benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -38,7 +43,12 @@ private BenchmarkServiceGrpc() {}
               io.grpc.benchmarks.proto.Messages.SimpleRequest.getDefaultInstance()))
           .setResponseMarshaller(io.grpc.protobuf.ProtoUtils.marshaller(
               io.grpc.benchmarks.proto.Messages.SimpleResponse.getDefaultInstance()))
+          .setMethodOptions(SERVICE_PROTO_DESCRIPTOR
+              .getMethodDescriptor(METHODIDX_UNARY_CALL).getOptions())
```

Comment:
I'm not sure I understand why? Even if the protobuf is generated separately, the indexes should remain the same - if they're different it'll mean the files are different and you have much bigger problems because it'll mean that the files are out-of-sync. I think I might be missing something, please explain.